hbase10/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java

```diff
@@ -166,7 +176,9 @@ public void init() throws DBException {
     String table = com.yahoo.ycsb.workloads.CoreWorkload.table;
     try {
       final TableName tName = TableName.valueOf(table);
-      connection.getTable(tName).getTableDescriptor();
```

Comment:
I thought Connection was supposed to be threadsafe? Is this essentially a bug in the implementation? Any idea if it still exists in later HBase client versions?

I'm happy to demonstrate a workaround other applications need to be aware of, but I'd like to make sure we document why we're being more conservative than the HBase docs indicate we need to be.